specification/sql/resource-manager/Microsoft.Sql/preview/2021-02-01-preview/LedgerDigestUpload.json

```diff
@@ -0,0 +1,331 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-02-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/ledgerDigestUpload/{ledgerName}": {
+      "get": {
```

Comment:
>   "get": { [](start = 3, length = 11)

Is there a delete operation for this resource type?

Reply:
no, the way to turn it off is via the /disable operation

Reply:
The user should be able to delete the resource. Here's RPC documentation about Proxy resource deletes: https://github.com/Azure/azure-resource-manager-rpc/blob/2e1f2c11ce4cfdf309f9cc8ae71575cb8fb83319/v1.0/Addendum.md#designing-resources

Reply:
In this scenario this does not make sense, there is no such thing as 'deleting' this resource since it is an endpoint configuration of where to upload information. 
We do not want to expose a delete operation and have users think that their previous information will be deleted or the storage account will be deleted

Reply:
Remind me. Do you have a create API for this or does the resource always exist?

Reply:
There is an update API, but no "create" since we are just updating a setting

Reply:
It looks like this should really be a POST action (to enable and disable ledgerDigestUpload)on the database resource. @majastrz what do you think?

Reply:
One of implications of POST you may want to consider is that Azure policy does not support it. If you have any scenarios where the user wants to have a policy reject disabling the upload configuration, enable/disable should be done via PATCH.